src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculate from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Based on the requirements of this project, we are supposed to make the App component class-based. Kindly fix :wrench: this.

Reply:
Please can you explain further because when I use class-based component in the app.js, I get this linters error saying 
`Component should be written as a pure function  react/prefer-stateless-function`